10up/distributor
# Fix/update admin icon

### Description of the Change


Closes #1119

Updates the admin SVG icon to have roughly the same width & height so it no long appears squished in the admin menus or in the block editor plugin. The main benefit realized is that it will no longer mildly annoy me whenever I notice it's squished 😉 

The SVG has been reworked & simplified to smooth out several issues that appeared when the original was enlarged. It's down to just 2 path elements which reduces file size greatly (8KB to 1.5KB) and it should make it somewhat easier to update in the future.

Applies to:
- Main admin menu plugin icon
- Pages/Posts list plugin column icon
- Block editor plugin icon
- README (icon at the top, when merged into trunk)

Examples of how it looked before vs with the updated icon:

| Before  | Updated |
| --- | --- |
| ![old-icon](https://github.com/10up/distributor/assets/6644396/1074f627-f0af-4d63-bc38-765b838981df) | <img src="https://user-images.githubusercontent.com/6644396/266190095-ef895c4a-ce59-4f3b-b495-8202b4d8ee15.svg" width="36" height="36"> |
| <img width="158" alt="2023-09-06_20-52-3LOLjpfR" src="https://github.com/10up/distributor/assets/6644396/223e87a7-f801-44f5-8a14-429ae6f083fe">  | <img width="160" alt="2023-09-06_20-53-KP7F9TLm" src="https://github.com/10up/distributor/assets/6644396/2e435cae-14b3-4cb1-bbd3-b15f72a90489"> |
| <img width="160" alt="2023-09-06_20-54-Jz3aw2Ri" src="https://github.com/10up/distributor/assets/6644396/8106f541-0c52-4246-8557-37a2761cee89"> | <img width="160" alt="2023-09-06_20-54-XzgYtqjI" src="https://github.com/10up/distributor/assets/6644396/f34e84d3-8848-4409-8b8b-aac2c180b398"> |
| <img width="50" alt="2023-09-06_20-55-3IA3Vm8h" src="https://github.com/10up/distributor/assets/6644396/bfdbb3fc-5562-4e44-b6bf-c046973ac86f"> | <img width="51" alt="2023-09-06_20-55-YxvIwLWi" src="https://github.com/10up/distributor/assets/6644396/1df14753-d793-4092-a739-a2742963fe90"> |
| <img width="283" alt="2023-09-06_20-58-twzCj9ad" src="https://github.com/10up/distributor/assets/6644396/bc337580-f4e6-4c9e-80aa-2657f90a75b6"> | <img width="287" alt="2023-09-06_20-57-WqLyGegZ" src="https://github.com/10up/distributor/assets/6644396/6a8c4eaa-4491-4284-981d-e02b802d9a63"> |

### How to test the Change
1. Install and activate Distributor plugin
2. Check admin main menu "Distributor" icon is updated & showing correctly
3. Check that the updated menu icon color matches other admin menu icons including on hover & when it's active while visiting a Distributor setting sub-page (should auto-update to white when active)
4. Check Posts/Pages list page & verify the Distributor column icon is updated & showing correctly
5. Open a Post/Page block editor and verify the icon in the Distributor tab in the sidebar page settings is updated  & showing correctly

### Changelog Entry
> Fixed - Admin icon has been updated so it has the same width/height

### Credits
Props @zach-adams

### Checklist:
